numpy/core/include/numpy/npy_3kcompat.h

```diff
@@ -193,7 +201,7 @@ npy_PyFile_Dup2(PyObject *file, char *mode, npy_off_t *orig_pos)
     /* Record the original raw file handle position */
     *orig_pos = npy_ftell(handle);
     if (*orig_pos == -1) {
-        PyErr_SetString(PyExc_IOError, "obtaining file position failed");
+        PyErr_SetString(PyExc_IOError, "Obtaining file position failed");
```

Comment:
Typically error messages are not capitalized, because they come directly after, e.g.`IOError: obtaining...`